src/js/contentscripts/dnt.js

```diff
@@ -26,22 +26,28 @@ function getPageScript() {
       OBJECT.defineProperty(OBJECT.getPrototypeOf(NAVIGATOR), "doNotTrack", {
         get: function doNotTrack() {
           return "1";
-        }
+        },
+        configurable: true,
+        enumerable: true
       });
     }
 
-    if (!NAVIGATOR.globalPrivacyControl) {
+    if (!globalThis.navigator.globalPrivacyControl) {
```

Comment:
This should stay as `NAVIGATOR`. We (A) want to use the previously saved object reference, and (B) don't want to use `globalThis` as previously discussed.

Reply:
oops, I forgot about that and completely missed this part